operator/src/main/helm/coherence/README.md

```diff
@@ -49,6 +49,7 @@ Coherence 12.2.1.4.
 | `cluster` | Cluster name of Coherence cluster member | Helm release name |
 | `serviceAccountName` | Name of the service | `"default"` |
 | `store.cacheConfig` | Name of the cache configuration file | |
+| `store.mainClass` | Coherence container main class | com.tangosol.net.DefaultCacheServer |
```

Comment:
startCoherence.sh provides command line options for a set of predefined mainClass (such as console, DefaultCacheServer).  Does there need to be a way to provide arguments to the store.mainClass with a property like store.mainClass.arguments?

Reply:
I don't see the need to pass arguments to main class since all the "arguments" for Coherence can be specified through properties.

Reply:
Discussed with Harvey.  Added store.mainClassArguments per your suggestions.